app/views/static_pages/support.html.haml

```diff
@@ -8,3 +8,8 @@
 
 %h3 Make a donation
 = render 'donate'
+
+%h3 Set up AmazonSmile
+
+%p
+  You can also set up #{external_link_to "AmazonSmile", "http://smile.amazon.com/"} to automatically give Double Union 0.5% of the purchase price of your Amazon purchases (with some exceptions). When you visit AmazonSmile, it'll ask you to pick a non-profit organization; you can search for "Double Union".
```

Comment:
@brittag  `When you visit AmazonSmile, it'll ask you to pick a non-profit organization; you can search for "Double Union".` has a period outside the the quotation marks which is technically ungrammatical, but easier to copy, paste, and search. Could we change it to something like `When you visit AmazonSmile, search for "Double Union" when prompted to pick a non-profit organization.` to avoid the grammar conflict? Let me know if this is too nitpicky!